auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -37,11 +48,95 @@
                 <div>{{ displayStatus(item.status) }}</div>
               </template>
 
-              <!-- Displaying Formatted Date -->
+              <!-- Displaying Formatted Submission Date -->
               <template #[`item.submissionDate`]="{ item }">
                 <div>{{ formatDate(item.submissionDate) }}</div>
               </template>
 
+              <!-- Displaying Formatted Future Effective Date and Tooltips-->
+              <template #[`item.futureEffectiveDate`]="{ item }">
+                <div>
+                  <v-tooltip bottom>
```

Comment:
Can you use the same tooltip as elsewhere (which appears above the item and has a little pointer)?

Reply:
I see why you did this: it's to show the icon tooltip above and the date tooltip below. That's OK with me.

If you keep this, I'd like to see the little arrow, ie:

![image](https://github.com/user-attachments/assets/0b7ef5d6-c954-425c-bbd3-f0b392eece1b)

Reply:
Sure, I will update it